odoo/addons/base/models/res_users.py

```diff
@@ -1796,6 +1796,34 @@ def fields_get(self, allfields=None, attributes=None):
             })
         return res
 
+    def action_change_password(self):
+        if len(self) == 1:
+            user_id = self.env['change.password.user'].create({
+                'user_id': self.id,
+                'user_login': self.login,
+            })
```

Comment:
Instead of manually creating the record, you could pass the values in the action's context as `default_XXX` :innocent: